app/models/forum.rb

```diff
@@ -0,0 +1,3 @@
+class Forum < ApplicationRecord
+    has_and_belongs_to_many :users
```

Comment:
I'm guessing this came from a tutorial but just wanted to let you know I don't see this too often in the projects I work on. Usually there is a join model in between the 2 tables. So in this case there would be another model called `ForumUsers` or something and each model (`Forum` and `User`) would `have_many :forum_users`. 

It is ok if that doesn't make sense right now, just wanted to point it out that you might not see this very often :)